tests/test_hooks.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+import pytest
+
+from requests import hooks
+
+
+def hook(value):
+    return value[1:]
+
+
+@pytest.mark.parametrize(
+    'hooks_list, result', (
+        (hook, 'ata'),
+        ([hook, lambda x: None, hook], 'ta'),
+    )
+)
+def test_hooks(hooks_list, result):
```

Comment:
So, this test is now pretty opaque: looking at the parametrized results don't make it enormously clear what it's supposed to be testing. A docstring or comment that explains how this test should function would be good.